### PR TITLE
feat(index.ts): Parse the first 10 lines of each file for flow pragma

### PR DESCRIPTION
**Pull request by kevinSuttle**
Flow itself only checks the first 10 lines when looking for the `@flow` pragma. This commit mimicks
that behavior. Fixes https://github.com/withspectrum/danger-plugin-flow/issues/3

This commit also adds test files for future validation.

### CODE REVIEW

**Comment by mxstbr** [comment]
Yesss, thank you so much! 💜

**Comment by kevinSuttle** [comment]
Of course! Thanks for having me. :)

File: src/index.ts [hunk 1]
```diff
@@ -47,7 +47,15 @@ export default async function flow(options: Options = {}) {
     )
   )
 
-  const unflowedFiles = files.filter(({ content }) => !/@flow/.test(content)).map(({ path }) => path)
+  function detectFlowPragma(content): boolean {
+    const flowPragma = "@flow"
+    const maxLines = 10
```
Comment (iamdustan) [inline src/index.ts]: nit: this should read from `.flowconfig -> [options] -> max_header_tokens` https://flow.org/en/docs/config/options/#toc-max-header-tokens-integer and default to 10.

Comment (mxstbr) [reply src/index.ts]: Ohh that's a good idea, we should check whether the flowconfig has that option set for maximum compatibility!

Comment (kevinSuttle) [reply src/index.ts]: Wow, I had no idea. Thanks, Dustan!

Comment (kevinSuttle) [reply src/index.ts]: https://repl.it/@kevinSuttle/Parsing-flowconfig-to-find-maxheadertokens

**Pull request merged**